pkg/cmd/server/origin/master.go

```diff
@@ -128,7 +128,14 @@ func (c *MasterConfig) Run(kubeAPIServerConfig *kubeapiserver.Config, assetConfi
 	}
 	// TODO this goes away in 3.7 after we commit to the aggregator always being on (even if its just in local mode).
 	// this is installing the openshift APIs into the kubeapiserver
+	// ok, this is a big side-effect.  Openshift APIs run a different admission chain (always have), but since
+	// we're going through a "normal" API installation in the wrong server, we need to switch the admission chain
+	// *only while we're installing these APIs*.  There are tests that make sure this works and doesn't drop
+	// plugins and we'll remove it once we're aggregating
+	kubeAPIServer.GenericAPIServer.SetAdmission(openshiftAPIServerConfig.GenericConfig.AdmissionControl)
```

Comment:
ugly, but ok.

Wondering which of these temporary hacks we can move into the if-clause below for the non-aggregating run mode?